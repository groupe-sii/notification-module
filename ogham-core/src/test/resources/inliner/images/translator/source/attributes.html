<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
</head>
<body>
	<img src="images/h1.gif" data-inline-image="skip" />
	<img src="images/fb.gif" data-inline-image="base64" />
	<img src="images/tw.gif" data-inline-image="attach" />
	<img src="images/right.gif" data-image-inlined="true" />
</body>
</html>