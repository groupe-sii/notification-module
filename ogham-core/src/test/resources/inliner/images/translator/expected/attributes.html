<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
 <head> 
 </head> 
 <body> 
  <img src="images/h1.gif"> 
  <img src="data:image/gif;base64,R0lGODlhJgAmANUAAMs8QLMxNeNHS+pKTuFGSrw1OdtDR7YyNudJTbIwNLEwNO1MULAwNMg7P+1LT7AvM7s0ONhCRuxLT8Q5PelKTtxESLUyNutLT782OuVITLgzN99FSehJTbk0OMk7P9lCRsI4PMo7P9A+QrczN+RITNZBRcc6Pr83O701OcE3O9RARMY6PuZITN5ESOBGSr42OtpDR7kzN9dBRbo0OM09QcM4PNI/Q9M/Q8A3O9NARORHS91ESLQxNd5FSa8vM+5MUCH/C1hNUCBEYXRhWE1QPD94cGFja2V0IGJlZ2luPSLvu78iIGlkPSJXNU0wTXBDZWhpSHpyZVN6TlRjemtjOWQiPz4gPHg6eG1wbWV0YSB4bWxuczp4PSJhZG9iZTpuczptZXRhLyIgeDp4bXB0az0iQWRvYmUgWE1QIENvcmUgNS4wLWMwNjAgNjEuMTM0Nzc3LCAyMDEwLzAyLzEyLTE3OjMyOjAwICAgICAgICAiPiA8cmRmOlJERiB4bWxuczpyZGY9Imh0dHA6Ly93d3cudzMub3JnLzE5OTkvMDIvMjItcmRmLXN5bnRheC1ucyMiPiA8cmRmOkRlc2NyaXB0aW9uIHJkZjphYm91dD0iIiB4bWxuczp4bXA9Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC8iIHhtbG5zOnhtcE1NPSJodHRwOi8vbnMuYWRvYmUuY29tL3hhcC8xLjAvbW0vIiB4bWxuczpzdFJlZj0iaHR0cDovL25zLmFkb2JlLmNvbS94YXAvMS4wL3NUeXBlL1Jlc291cmNlUmVmIyIgeG1wOkNyZWF0b3JUb29sPSJBZG9iZSBQaG90b3Nob3AgQ1M1IE1hY2ludG9zaCIgeG1wTU06SW5zdGFuY2VJRD0ieG1wLmlpZDo1QjcwQTA5RkJFMTUxMUUyQjUwQ0YzRjY0Nzg2NURGMCIgeG1wTU06RG9jdW1lbnRJRD0ieG1wLmRpZDo1QjcwQTBBMEJFMTUxMUUyQjUwQ0YzRjY0Nzg2NURGMCI+IDx4bXBNTTpEZXJpdmVkRnJvbSBzdFJlZjppbnN0YW5jZUlEPSJ4bXAuaWlkOjVCNzBBMDlEQkUxNTExRTJCNTBDRjNGNjQ3ODY1REYwIiBzdFJlZjpkb2N1bWVudElEPSJ4bXAuZGlkOjVCNzBBMDlFQkUxNTExRTJCNTBDRjNGNjQ3ODY1REYwIi8+IDwvcmRmOkRlc2NyaXB0aW9uPiA8L3JkZjpSREY+IDwveDp4bXBtZXRhPiA8P3hwYWNrZXQgZW5kPSJyIj8+Af/+/fz7+vn49/b19PPy8fDv7u3s6+rp6Ofm5eTj4uHg397d3Nva2djX1tXU09LR0M/OzczLysnIx8bFxMPCwcC/vr28u7q5uLe2tbSzsrGwr66trKuqqainpqWko6KhoJ+enZybmpmYl5aVlJOSkZCPjo2Mi4qJiIeGhYSDgoGAf359fHt6eXh3dnV0c3JxcG9ubWxramloZ2ZlZGNiYWBfXl1cW1pZWFdWVVRTUlFQT05NTEtKSUhHRkVEQ0JBQD8+PTw7Ojk4NzY1NDMyMTAvLi0sKyopKCcmJSQjIiEgHx4dHBsaGRgXFhUUExIREA8ODQwLCgkIBwYFBAMCAQAAIfkEAAAAAAAsAAAAACYAJgAABv/An3BILP4WBIKL4DA6n85BpJbwWa8KUIkC7Q4Rpqt4bJ1kvM4LQCFOFCaNxqQQEDM8A/SQoLkmGgZNRQsVIXVWBxt6FWw+Dx5cXgNrVgwfXi0MVhoCekMZM1YPMFA6jRh5nkMXKZUETg4dViiCqkMLOFYjqUQiiJFdJEm8QwN9PjRFA4cuXjRVVgZOAg8+CQhEAGVeJWPSTg1WJkMSdQ9nXQVXHQWKTghsCqkGViBoh99dK1YyQh5WEe5Z6eSFno8JQtT54CDQB8EuF6qNEFIlQBeDZCp0OSZhgJUCF8lYYdGllQ8ECKzgCEkmgZcJAzlYwdBlQBJoHwg8fALCConGC1Yg6Dm0EwoGK9gsWBs6UM8BHwwW/DjR1AtRNCl9CP1Bw4qNhkWddPMBQAiBj2DRUPWhUUiMaFarQtlgxYJUISqC3n1ytcsLKyKISHjq40aXvk8iWAlA7IdiqD1sGSHQSIWTMD54kJD8hXCNJw7++gjQgjMBpVolQKHw1pGIWl4W3NDk4wC2mkd1BfQCQ9ZHhmgchBBjARCwHwN2hCBsZYVqVRsUjjnAXAyEtpw/oBA5ZkY/zkVY5GhQQCGEAg1soAPPvksQADs="> 
  <img src="cid:0"> 
  <img src="images/right.gif">  
 </body>
</html>